scio-extra/src/main/scala/com/spotify/scio/extra/sparkey/ShardedSparkeyUri.scala

```diff
@@ -39,14 +38,12 @@ trait ShardedSparkeyUri extends SparkeyUri {
   private[sparkey] def exists: Boolean
   override def toString: String = basePath
 
-  implicit def coderSparkeyURI: Coder[ShardedSparkeyUri] = Coder.kryo[ShardedSparkeyUri]
-
   def basePathForShard(shardIndex: Short, numShards: Short): String =
     f"$basePath/part-$shardIndex%05d-of-$numShards%05d"
 
   def sparkeyUriForShard(shardIndex: Short, numShards: Short): SparkeyUri
 
-  val globExpression: String = s"$basePath/part-*"
+  lazy val globExpression: String = s"$basePath/part-*"
```

Comment:
You must add `transient` to avoid serialization (I think that was the goal, right ?)